docs/src/app/(docs)/uploading-files/page.mdx

```diff
@@ -99,16 +99,61 @@ Once your backend adapter has received and validated the request, you will next
 need to generate presigned URLs. First, generate some file keys for the files to
 be uploaded.
 
-To generate a file key, generate a sha256 hex digest of a file seed of your
-choice, as well as of your API key. Concatenate the API key and the file seed
-with a `:` separator and encode the result as a base64 string.
+To generate a file key, generate a [Sqids](https://sqids.org/) id of a file seed
```

Comment:
The file seed doesn't technically need to be a sqids right? We only do verification on the appId seed part